odtlearn/RobustTree.py

```diff
@@ -213,24 +212,52 @@ def print_tree(self):
         check_is_fitted(self, ["model"])
 
         assignment_nodes = []
-        for n in self.model.tree.Nodes + self.model.tree.Leaves:
+        for n in self.grb_model.tree.Nodes + self.grb_model.tree.Leaves:
             print("#########node ", n)
             terminal = False
 
             # Check if pruned
-            if self.model.tree.get_parent(n) in assignment_nodes:
+            if self.grb_model.tree.get_parent(n) in assignment_nodes:
                 print("pruned")
                 assignment_nodes += [n]
                 continue
 
-            for k in self.model.labels:
+            for k in self.grb_model.labels:
                 if self.w_value[n, k] > 0.5:
                     print("leaf {}".format(k))
                     terminal = True
                     assignment_nodes += [n]
                     break
             if not terminal:
-                for (f, theta) in self.model.f_theta_indices:
+                for (f, theta) in self.grb_model.f_theta_indices:
                     if self.b_value[n, f, theta] > 0.5:  # b[n,f]== 1
                         print("Feature: ", f, ", Cutoff: ", theta)
                         break
+
+    def plot_tree(
+        self,
+        label="all",
+        filled=True,
+        rounded=False,
+        precision=3,
+        ax=None,
+        fontsize=None,
+        color_dict={"node": None, "leaves": []},
+    ):
+        check_is_fitted(self, ["model"])
+        exporter = MPLPlotter(
+            self.grb_model,
+            self.X_col_labels,
+            self.b_value,
+            self.w_value,
+            self.p_value,
```

Comment:
There is no `self.p_value` in Robust trees, but you can obtain it by summing over the labels `k` from `self.w_value` . If you need to implement the plotting function with `self.p_value`, let me know.